### PR TITLE
Some minor usability enhancements for pyboard.py

### PR DESCRIPTION
**Pull request by larsks**
- Allow `-d` as an alias for `--device`
- Allow defaults for `--device` and `--baud` to be set via
  `PYBOARD_DEVICE` and `PYBOARD_BAUD` environment variables.

### CODE REVIEW

File: docs/reference/pyboard.py.rst [hunk 1]
```diff
@@ -59,6 +60,14 @@ with the device.::
     $ pyboard.py --device /dev/ttyACM0 -c 'print(1+1)'
     2
 
+If you are often interacting with the same device, you can set the environment
+variable ``PYBOARD_DEVICE`` as an alternative to using the ``--device``
+command line option.  For example, the following is equivalent to the previous
```
Comment (tve) [inline docs/reference/pyboard.py.rst]: This should mention `PYBOARD_BAUD` too

**Comment by dpgeorge** [comment]
Thanks for the contribution, this looks good.

Since the long-form option for the baudrate is called `--baudrate` I think the env variable for this should be called `PYBOARD_BAUDRATE`.

File: docs/reference/pyboard.py.rst (hunk 1 above)
Comment (larsks) [reply docs/reference/pyboard.py.rst]: Thanks, good catch. I've updated the docs.

**Comment by larsks** [comment]
@dpgeorge, thanks for the suggestion. I've updated `pyboard.py` to use `PYBOARD_BAUDRATE`.

**Comment by dpgeorge** [comment]
Thanks for updating, merged in 1cf994c48b912254802f48e31eaf95e49ac55a82 and 3a0f64fc7aafe9fa39f518aec389ef5f55b40007

**Pull request closed without merging**